docs/changelog/README.rst

```diff
@@ -0,0 +1,19 @@
+Examples for changelog entries adding to your Pull Requests
+
+file ``544.doc.rst``::
+
+    Update requirements file to replace
+    requirements.txt with requirements-build.in - by :user:`akasurde`
+
+file ``544.feature.rst``::
+
+    Added sftp get functionality to fetch file from remote host  - by :user:`amolkahat`
+
+file ``571.bugfix.rst``::
+
+    Fix double close issue, ddd logic to free
+    the channel allocated memory within
+    `__dealloc__() <finalization_method>`  - by :user:`ganeshrn`
+
+.. Note::
```

Comment:
I believe it shouldn't be capitalized but nevermind, I'm going to have this file included in my PR in a proper state anyway.